### DISCUSSION TITLE
Please consider compatibility with Blade Ui kit

### DISCUSSION THREAD

**Discussion by jcc5018** [livewire/flux#399] [Feature Requests]
Can you please consider making this more flexible with any icon package someone may use? 

[Blade Ui Kit](https://blade-ui-kit.com/blade-icons) is built with tailwinds in mind. I personally have my site set up with font awesome and icons already loaded in my project. They have far more options than heroicons, so I'd prefer to stick with that.  But if you can integrate with the package to accept the SVG icons, then people can use what they want.

**Comment by ju5t** [comment]
Being able to pass icons as slots would be nice. I think any other native implementation will be a slippery slope. People always want more or other icons to be supported.

Icon slots would solve all of that.
